src/rhtap/core/integration/ci/providers/githubActionsCI.ts

```
@@ -3,23 +3,23 @@ import {
 import { KubeClient } from '../../../../../api/ocp/kubeClient';
 import { PullRequest } from '../../git/models';
 import { BaseCI } from '../baseCI';
 import { CIType, EventType, Pipeline, PipelineStatus } from '../ciInterface';
 
 export class GitHubActionsCI extends BaseCI {
-  private githubAction!: GitHubActionsClient;
+  private githubClient: GithubClient;
   private componentName: string;
   private secret?: Record<string, string>;
-  private clientFactory: GitHubClientFactory;
+  private githubClient: GithubClient;
```

Biome (2.1.2)

[error] Duplicate class member name "githubClient"



(lint/suspicious/noDuplicateClassMembers)